ansible/pbTestScripts/qemuPlaybookCheck.sh

```diff
@@ -137,9 +166,11 @@ setupWorkspace() {
 		find "$workFolder" -type f | xargs rm -f
 		rm -rf "$workFolder"/openjdk-infrastructure "$workFolder"/openjdk-build
 	fi
-	if [[ ! -f "${workFolder}/${ARCHITECTURE}.dsk" ]]; then 
+	if [[ ! -f "$workFolder/$OS.$ARCHITECTURE.dsk" ]]; then 
```

Comment:
Why remove the braces around these variables?

Reply:
I must have done it to make is consistent with the changes I made elsewhere in the script. Ive just tested it and either way works.